requirements/constraints.txt

```diff
@@ -78,8 +77,7 @@ pylint<2.16.0 # greater version failing quality test. Fix them in seperate ticke
 # Deprecated version of the AWS SDK;
 # we should stop using this
 boto==2.39.0
```

Comment:
do we still need to constrain the boto version? or are we able to remove this now?

Reply:
> do we still need to constrain the boto version? or are we able to remove this now?

`django-storages` has no link with `boto`.